### ISSUE TITLE
split_closed_faces should default to False and not allow None

### ISSUE THREAD

**Issue by greg-kcio** [KittyCAD/kittycad.py#253]
`kittycad.models.input_format.step` and `kittycad.models.input_format.sldprt` each have a new property `split_closed_faces`. They are `Optional[bool]` so they should accept `True`, `False`, or `None`, but assigning `None` causes the engine to return an error and close the websocket connection:

```
{'error_code': <ErrorCode.INVALID_BSON: 'invalid_bson'>, 'message': 'invalid type: unit value, expected a boolean'}
```

**Comment by greg-kcio** [comment]
The error began after this PR: https://github.com/KittyCAD/kittycad.py/pull/246

**Comment by greg-kcio** [comment]
Fixed by #254
